hal/halx86/smp/i386/apentry.S

```diff
@@ -22,11 +29,12 @@ _APEntry:
     lidt cs:[idtdesc - _APEntry]
 #endif
 
-    mov   eax, cr0
-    or    eax, HEX(00000001)
-    mov   cr0, eax
+    mov eax, cr0
+    or  eax, HEX(00000001)
+    mov cr0, eax
```

Comment:
Small comment like "enable protected mode" maybe ? Also, there is CR0_PE constant.

Reply:
Are you asking me to redefine the CR0_PE constant? 
I'm not entirely sure how id pull that into the bootstub in a way an application processor can access it.

Reply:
no, I mean instead of `HEX(00000001)`, use `CR0_PE` from ks386.h